src/commands/bridge/test-routes.ts

```diff
@@ -0,0 +1,66 @@
+import { KeyringPair } from '@polkadot/keyring/types'
+import { GluegunToolbox, filesystem, print } from 'gluegun'
+import {
+  EthereumConfig,
+  Network,
+  SubstrateConfig,
+} from '@buildwithsygma/sygma-sdk-core'
+import { Wallet } from 'ethers'
+import { InitializedWallets, RpcEndpoints } from '../../types'
+
+import { testSubstrateRoutes } from '../../utils/substrate/test-routes'
+import { testEvmToEvmRoutes } from '../../utils/evm/testEVMToEVMRoutes'
+import { testEvmToSubstrateRoutes } from '../../utils/evm/testEVMToSubstrateRoutes'
+
+module.exports = {
+  name: 'test-routes',
+  run: async (toolbox: GluegunToolbox) => {
+    const { sharedConfig, wallet } = toolbox
+
+    const rawConfig = await sharedConfig.fetchSharedConfig()
+
+    const { env } = toolbox
```

Comment:
nit: destruct in same line as `sharedConfig` and `wallet`

Reply:
Then it will not work. The toolbox.env is set in fetchSharedConfig function

Reply:
Then it will not work. The toolbox.env is set in fetchSharedConfig function